examples/player/simid_player.html

```diff
@@ -35,11 +35,43 @@
       simidPlayer.initializeAd();
     }
 
+    function disableButtons(state) {
```

Comment:
I would rework the naming here a bit.
Since this can now either disable or enable the buttons, something like setButtonState might convey that better.  Then instead of state as the param value, you can use disable, since that will specifically convey what aspect of the state this is setting.

Reply:
sounds good!